packages/core/tests/solana-signer-swap.spec.ts

```diff
@@ -0,0 +1,361 @@
+import { afterEach, beforeAll, describe, expect, it } from '@jest/globals';
+import { Connection, Keypair, PublicKey, Signer } from '@solana/web3.js';
+import {
+  createScheduledNeonEvmMultipleTransaction,
```

Comment:
@oable, Why do we need two almost identical methods, `createScheduledNeonEvmMultipleTransaction` and `createScheduledNeonEvmTransaction`, with the only difference being the `ScheduledTransactionTag`? Wouldn't it be more convenient to pass the tag as an argument instead?

Reply:
Yes, I improved it, removed the duplicate code for the instruction, and added the `isMultible` attribute to the method data.